t/Path.t

```diff
@@ -800,7 +800,11 @@ is(
         "If not provided with any paths, make_path() will return a count of 0 things created");
 }
 
-{
+ SKIP: {
+    my $skip_count = 5;
+    skip "getpwuid() and getgrgid() not implemented on Windows", $skip_count
+        if $^O eq 'MSWin32';
+
```

Comment:
This is very close to what I developed yesterday in the correct-win32-tests-20170419 branch (https://github.com/jkeenan/File-Path/tree/correct-win32-tests-20170419).  (I was holding off merging it into master until I got feedback from my local colleague.)

But I think one (actually 2 x 1) thing should be changed.  The 'skip' message you're using is, I think, appropriate for the other test blocks in t/Path.t in which it is used, but not for these two.  We don't call either ```getpwuid()``` or ```getgrgid()``` in these two blocks.  Could we go with something like:
```
        skip "Windows will not set this error condition", $skip_count
            if $^O eq 'MSWin32';
```